seaborn/utils.py

```diff
@@ -173,9 +173,9 @@ def despine(fig=None, ax=None, top=True, right=True, left=False,
         Specific axes object to despine.
     top, right, left, bottom : boolean, optional
         If True, remove that spine.
-    offset : int or None  (default), optional
+    offset : int or dict of ints or None  (default), optional
```

Comment:
"dict of ints" does not make sense because the keys of the dict are not ints. Just say `dict` and provide more information below.